.github/actions/baas-test-server/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "@realm/baas-test-server-action",
```

Comment:
We could probably rename the start-server and stop-server scripts to use an .mjs extension and avoid this package.json. On the other hand, it could come in handy to define action specific dependencies if added as a workspace package too.